build/deploy/main.go

```diff
@@ -85,13 +85,13 @@ func uploadPlugin(client *model.Client4, pluginID, bundlePath string) error {
 	log.Print("Uploading plugin via API.")
 	_, resp := client.UploadPluginForced(pluginBundle)
 	if resp.Error != nil {
-		return fmt.Errorf("Failed to upload plugin bundle: %s", resp.Error.Error())
+		return errors.Wrap(resp.Error, "failed to upload plugin bundle")
 	}
 
 	log.Print("Enabling plugin.")
 	_, resp = client.EnablePlugin(pluginID)
 	if resp.Error != nil {
-		return fmt.Errorf("Failed to enable plugin: %s", resp.Error.Error())
+		return errors.Wrap(resp.Error, "Failed to enable plugin")
```

Comment:
Are we enforcing `errors.Wrap` instead of `fmt.Errorf`?

Reply:
Not, we are not enforcing this, but you are bringing up another great points.

The complain from `golangci-lint` for the original message is:
` error strings should not be capitalized or end with punctuation or a newline (golint)`, which is actually not fixed with the change. `golangci-lint` just doesn't find the capitalized message.

But given that `build` is just a copy from https://github.com/mattermost/mattermost-plugin-starter-template I leaning towards not fixing it in this PR.